packages/governance/test/swingsetTests/committeeBinary/test-committee.js

```diff
@@ -42,7 +42,7 @@ test.before(async t => {
   };
   const config = { bootstrap: 'bootstrap', vats };
   config.bundles = { zcf: { bundle: zcfBundle } };
-  config.defaultManagerType = 'xs-worker';
+  config.defaultManagerType = 'xs-worker'; // originally wanted for metering
```

Comment:
I'm not sure what the metering requirement is but I I think this line can be stricken.

Reply:
Yeah, I looked at the history for that file, and the setting was added in a commit (5108c51b73f28c86f06c90640c3f90265435b14a) where I said "use metered=true and xs-worker on all swingset tests", so I figured I'd capture that intent (however imprecise/ill-considered I might have been at the time). I wanted to cache the results of that lookup so we can feel more confident removing the `defaultManagerType` there in the future.

I left in-place the other `defaultManagerType` settings that I couldn't confidently understand.